spk/dotnet10-runtime/Makefile

```diff
@@ -0,0 +1,40 @@
+SPK_NAME = dotnet10-runtime
+SPK_VERS = 10.0.0-rc.2
```

Comment:
version number must contain only dots and digits, dash and alpha are not supported.

but we should not deploy release candiate versions anyway.

My intention is to wait for the official dotnet 10 release and drop dotnet 9 when dotnet 10 is released.